ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -129,6 +132,31 @@ func TestConvertToTaskDefinition(t *testing.T) {
 	}
 }
 
+func TestConvertToTaskDefinitionWithNoSharedMemorySize(t *testing.T) {
+	serviceConfig := serviceConfigWithDefaultNetworks()
+
+	taskDefinition := convertToTaskDefinitionInTest(t, "name", serviceConfig, "", "")
+	containerDef := *taskDefinition.ContainerDefinitions[0]
+
+	assert.Nil(t, containerDef.LinuxParameters.SharedMemorySize, "Expected sharedMemorySize to be null")
+}
+
+func TestConvertToTaskDefinitionWithSharedMemorySize(t *testing.T) {
+	// Realistically, we expect customers to specify sizes larger than the default of 64M
+	expectedMBs := 128
+	shmSize := yaml.MemStringorInt(int64(expectedMBs * miB))
```

Comment:
is there a way to add a test for the case where the yaml contains a string (e.g., '1gb') ?

Reply:
see `project_test.go`